student-store-express-api/routes/store.js

```diff
@@ -0,0 +1,27 @@
+const express = require("express");
+const { findAll } = require("../models/store");
+const Router = express.Router();
+const { findOne } = require("../models/store");
```

Comment:
Can combine this import statement with the other from `../models/store` by doing:

`const { findAll, findOne } = require("../models/store")`